src/components/marginals/Nav_Hero/Hero.style.jsx

```diff
@@ -120,10 +146,22 @@ export const Img2 = styled.img`
     left: 85px;
     top: 80px;
   }
-  @media (min-width: 1400px) {
```

Comment:
instead of using directly screen size here first add them in tailwind config file